edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -786,6 +787,10 @@ def __init__(self, element, remotes):
             self.tag = element.attrib['tag']
         except Exception:
             self.tag = None
+        try:
+            self.patchSet = element.attrib['patchSet']
+        except Exception:
+            self.patchSet = None

```

Comment:
To keep with the style precedent set elsewhere in this file please change self.patchSet to self.patch_set